.vscode-test.mjs

```diff
@@ -0,0 +1,12 @@
+import { defineConfig } from '@vscode/test-cli';
+
+export default defineConfig({
+	files: 'out/test/**/*.test.js',
+	workspaceFolder: './test Fixture with speci@l chars',
+	mocha: {
+		ui: 'tdd',
+		color: true,
+		timeout: 100000,
+		reporter: 'mocha-jenkins-reporter'
+	}
+});
```

Comment:
Interesting point: this could replace several of our runTests class in our other VS Code extensions.